MediaGalleryUi/view/adminhtml/web/js/action/deleteImageWithDetailConfirmation.js

```diff
@@ -72,22 +69,16 @@ define([
                 return '';
             }
             $.each(usedIn, function (entityName, count) {
-                message +=  count + ' ' +  this.getEntityNameWithPrefix(entityName, count) +  ', ';
+                message +=  count > 1 ? count + this.getEntityPrefix(entityName) : count + ' ' + entityName +  ', ';
             }.bind(this));
 
-            if (Object.keys(imageDetails).length  > 1) {
-                pronoun = 'These';
-                s = 's';
-                linkingVerb = 'are';
-            } else {
-                pronoun = 'This';
-                s = '';
-                linkingVerb = 'is';
-            }
-
+            entitiesCount = Object.keys(imageDetails).length  > 1;
             message = message.replace(/,\s*$/, '');
-            message = usedInMessage.replace('%s', message).replace('%n', pronoun).replace('%p', s)
-                .replace('%v', linkingVerb);
+            message = usedInMessage
```

Comment:
I don't think these changes would be helpful for translation as they are targeting English only.
i.e. in Russian locale the message will look like `These картинкаs are используется в 2 Катешорияs`.

Let's probably provide a generic message for both single and multiple assets: 

`
    The selected assets are used for the following entities content: Categories(2), Products(1), Pages(10).
`
Entity names should be translated separately.

@tkacheva @lenaorobei @sidolov what do you think about such approach?

Reply:
@sivaschenko Maybe, let's use the same approach on View Details?
![1_bloc_ex](https://user-images.githubusercontent.com/45624059/88913351-6704fa80-d269-11ea-9ac1-51d54372f220.png)
 Instead of 
![1_bloc_ar](https://user-images.githubusercontent.com/45624059/88913361-6cfadb80-d269-11ea-9510-aa425c23665c.png)